raco/backends/myria/myria.py

```diff
@@ -1481,14 +1470,15 @@ def fire(self, op):
 
 class PushIntoSQL(rules.Rule):
 
-    def __init__(self, dialect=None):
+    def __init__(self, dialect=None, push_grouping=False):
```

Comment:
Frankly I'm more concerned at this point about memory usage than performance, but being conservative is fine.